Descent3/Descent3.rc.in

```diff
@@ -0,0 +1,38 @@
+#define APSTUDIO_READONLY_SYMBOLS
+#include "winres.h"
+#undef APSTUDIO_READONLY_SYMBOLS
+
+IDI_ICON1 ICON DISCARDABLE "Descent3.ico"
+
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION 1,5,0,0
+ PRODUCTVERSION 1,5,0,0
```

Comment:
These two version entries might need the @ replacement, too.

Reply:
yep, fixed